tests/functional/r/recursion/recursion_error_3152.py

```diff
@@ -2,6 +2,6 @@
 import setuptools
 
 
-# pylint: disable=missing-docstring,too-few-public-methods
+# pylint: disable=missing-docstring,too-few-public-methods,abstract-method
```

Comment:
This unrelated change was already backported. Let's just get it in and get on with things.